.github/workflows/bm_maintenance.yml

```diff
@@ -1,6 +1,12 @@
 name: bare metal maintenance
 
 on:
+  workflow_call:
+    inputs:
+      containerd:
+        description: "Cleanup containerd store"
+        required: true
+        type: boolean
```

Comment:
This is not used, containerd still checks for the cron tab. If we run the containerd cleanup every night anyway, I suggest to remove the input and the condition on the job altogether.